packages/calcite-components/src/components/modal/modal.scss

```diff
@@ -3,26 +3,35 @@
  *
  * These properties can be overridden using the component's tag as selector.
  *
- * @prop --calcite-modal-content-background: Specifies the background color of content placed in the component's `"content"` slot.
- * @prop --calcite-modal-content-padding: Specifies the padding of the component's `"content"` slot.
- * @prop --calcite-modal-scrim-background: Specifies the background color of the component's scrim.
- * @prop --calcite-modal-width: Specifies the width of the component, using `px`, `em`, `rem`, `vw`, or `%`. Will never exceed the width of the viewport. Will not apply if `fullscreen` if set.
+ * @prop --calcite-modal-accent-color: Specifies the accent color of the component.
+ * @prop --calcite-modal-background-color: Specifies the background color of the component.
+ * @prop --calcite-modal-border-color: Specifies the border color of the component.
+ * @prop --calcite-modal-close-button-background-color-hover: Specifies the background color of the close button when hovered.
+ * @prop --calcite-modal-close-button-background-color: Specifies the background color of the close button.
+ * @prop --calcite-modal-close-button-icon-color: Specifies the icon color of the close button.
+ * @prop --calcite-modal-content-background-color: Specifies the background color of content placed in the component's `"content"` slot.
+ * @prop --calcite-modal-content-background: [Deprecated] Use `--calcite-modal-content-background-color` instead – Specifies the background color of content placed in the component's `"content"` slot.
+ * @prop --calcite-modal-content-padding: [Deprecated] Use `--calcite-modal-content-space` instead – Specifies the padding of the component's `"content"` slot.
```

Comment:
The `--calcite-modal-content-space` prop isn't documented.

Reply:
![epic-facepalm](https://github.com/Esri/calcite-design-system/assets/197440/09463a3e-28a5-4d86-8642-30be0cb07430)

Reply:
Fixed. Thanks for the catch! 🎣